samcli/lib/utils/git_repo.py

```diff
@@ -70,6 +70,9 @@ def _ensure_clone_directory_exists(clone_dir: Path) -> None:
             LOG.warning("WARN: Unable to create clone directory.", exc_info=ex)
             raise
 
+    def get_git_executable(self):
```

Comment:
is this method just to "public"-tize `_git_executable`?

Reply:
Yes.. I did this while testing to avoid updating other references to `_git_executable` but I can just change that method to be public.